src/app/core-ui/main/status/modal/help-modal/console-modal.component.ts

```diff
@@ -23,10 +23,14 @@ export class ConsoleModalComponent implements OnInit, AfterViewChecked {
 
   @ViewChild('debug') private commandContainer: ElementRef;
   log: any = Log.create('app-console-modal');
+
   public commandList: Command[] = [];
+  public commandHistory: Array<string> = [];
   public command: string;
   public currentTime: string;
   public disableScrollDown: boolean = false;
+  public isRpc: boolean = true;
```

Comment:
please change this variable name to `waitingForRPC` or something appropriate. `isRpc` is unclear

Reply:
sure